search-sentiments-map/public/style.css

```diff
@@ -31,3 +31,30 @@ body {
 .hidden {
   display: none;
 }
+
+#legend { 
+  display: flex;
+  display: -webkit-box;
+  height: 45px;
+  left: 110px;

```

Comment:
Having the legend to the top-left of the map seems a bit weird to me. Is it possible to have it overlay the map?